src/components/TermsOfUse/Notifications.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { useNavigate } from 'react-router';
+import termsofuseLogo from '../../images/termsofuse-logo.png';
+import { MdOutlineArrowForwardIos } from 'react-icons/md';
+import { MdFavoriteBorder } from 'react-icons/md';
+import FooterForRestoran from '../Restorans/FooterForRestoran/FooterForRestoran';
+const TermsOfUse = () => {
```

Comment:
Why is component `TermsOfUse` inside `Notifications.jsx` file? Please rename the file